doc/source/getting_started/docker/linux_container.rst

```diff
@@ -0,0 +1,178 @@
+   in the GitHub documentation.
+
+#. Save the token to a file with this command:
+
+   .. code-block:: bash
+
+       echo XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX > GH_TOKEN.txt
+
+#. Authorize Docker to access the repository. To see the commands to run, click the tab for your OS.
+
+   .. tab-set::
+
+       .. tab-item:: Linux/Mac
+
+           .. code-block:: bash
+
+               GH_USERNAME=<my-github-username>
+               cat GH_TOKEN.txt | docker login ghcr.io -u $GH_USERNAME --password-stdin
+
+       .. tab-item:: Powershell
+
+           .. code-block:: pwsh
+
+               $env:GH_USERNAME=<my-github-username>
+               cat GH_TOKEN.txt | docker login ghcr.io -u $env:GH_USERNAME --password-stdin
+
+       .. tab-item:: Windows CMD
+
+           .. code-block:: bash
+
+               SET GH_USERNAME=<my-github-username>
+               type GH_TOKEN.txt | docker login ghcr.io -u %GH_USERNAME% --password-stdin
+
+
+#. Pull the Geometry service locally using Docker with a command like this:
+
+   .. code:: bash
+
+      docker pull ghcr.io/ansys/geometry:linux-latest
+
+.. _ref_linux_docker_fromscratch:
+
+Building the Geometry Service Linux container
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Geometry service Docker containers can be easily built by following
+these steps.
+
+Inside the repository's ``docker`` folder, there are two ``Dockerfile`` files:
+
+* ``Dockerfile.linux``: this file builds the Linux-based Docker image.
+* ``Dockerfile.windows``: this file builds the Windows-based Docker image.
+
+Depending on the characteristics of the Docker engine installed in your
+machine, either one or the other has to be built.
+
+In this guide, focus is set on building the ``Dockerfile.linux`` image.
+
+Prerequisites
+~~~~~~~~~~~~~
+
+* Ensure that ``docker`` is installed in your machine.
+  If you do not have ``docker`` available, please refer to
+  :ref:`Running Linux Docker containers <ref_running_linux_containers>`.
+
+* Download the `latest Linux Dockerfile <https://github.com/ansys/pyansys-geometry/blob/main/docker/Dockerfile.linux>`_.
+
+* Download the `latest release artifacts for the Linux
+  Docker container (ZIP file) <https://github.com/ansys/pyansys-geometry/releases/latest/download/linux-binaries.zip>`_.
+
+* Move this ``.zip`` file to the location of the Linux Dockerfile previously downloaded.
+
+Building the Docker images
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to build your image, follow the next instructions:
+
+#. Locate yourself at the folder where the ``.zip`` file and the Dockerfile are placed.
+#. Run the following Docker command:
+
+  .. code:: bash
+
+     docker build -t ghcr.io/ansys/geometry:linux-latest -f Dockerfile.linux .
+
+#. Check that the image has been created successfully. You should see an output similar
+   to this one when running the following command:
```

Comment:
```suggestion
   to this one:
```
Isn't the code block showing the output from the previous command? This is what I thought--and why I removed the text that I did.